.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "src/Roslyn"]
+	path = src/Roslyn
+	url = https://github.com/nguerrera/Roslyn
+[submodule "src/CoreFx"]
+	path = src/CoreFx
+	url = https://github.com/nguerrera/corefx
```

Comment:
See https://github.com/dotnet/corefx/compare/master...nguerrera:il-analysis for changes to corefx depended upon so far. (Includes API-review pending official signature decoder 1.2 feature).